nmdc_automation/re_iding/scripts/re_id_tool.py

```diff
@@ -539,6 +542,14 @@ def delete_old_records(ctx, old_records_file):
     except requests.exceptions.RequestException as e:
         logging.error(f"An error occured while running: {delete_annotation_query}, response retutrned: {e}")
 
+    # write the deleted records to a tsv file
+    deleted_records_file = DATA_DIR.joinpath(f"{old_base_name}_deleted_records.tsv")
+    logging.info(f"Writing {len(deleted_records)} deleted records to {deleted_records_file}")
+    with open(deleted_records_file, "w") as f:
+        f.write("collection_name\ttype\tid\n")
+        for record in deleted_records:
```

Comment:
Assuming "record," here, is referring to a deleted Mongo document, I'd prefer the variable name be `record_identifier` or `record_descriptor`, since the variable doesn't contain the Mongo document.

Reply:
Refactored to `deleted_record_identifiers` and `record_identifiers`